cli/src/psp-utils/init.ts

```diff
@@ -32,13 +32,13 @@ export const initRepo = async (name: string, type: ProjectType) => {
       "generate",
       "--git",
       "https://github.com/Lightprotocol/psp-template",
-      // TODO(vadorovsky): Switch back to a new release when accounts macro is
-      // supported.
-      //
+      // TODO(vadorovsky): Switch back to a new release when
+      // https://github.com/Lightprotocol/psp-template/pull/12
+      // is merged and released.
       // "--tag",
       // PSP_TEMPLATE_TAG,
       "--branch",
-      "main",
+      "vadorovsky/rm-transaction-config",
```

Comment:
maiiiiin :D

Reply:
https://github.com/Lightprotocol/psp-template/pull/12 is merged

Reply:
Done